website/src/pages/en/token-api/token-api-faq.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Token API FAQ
+---
+
+## Table of Contents
+
+- [Basic Topics](#basic-topics)
+- [MCP / LLM / AI Topics](#mcp--llm--ai-topics)
+- [Advanced Topics](#advanced-topics)
+  - [Authentication & Access](#authentication--access)
+  - [Network Support](#network-support)
+  - [Query Parameters & Pagination](#query-parameters--pagination)
+  - [Response Format & Data Handling](#response-format--data-handling)
+  - [Common Issues & Troubleshooting](#common-issues--troubleshooting)
+  - [Integration & Usage](#integration--usage)
+
+## General
+
+### What blockchains does the Token API support?
+
+Currently, the Token API supports Ethereum, Binance Smart Chain (BSC), Polygon, Optimism, Base, and Arbitrum One.
+
+### Why isn't my API key from thegraph.market working?
+
+Make sure you are using the Access Token, not the API key. You can generate an access token from the dashboard on thegraph.market, using the dropdown menu next to each API key.
```

Comment:
I feel like this paragraph could make even more clear that Access Tokens are generated/derived from API Keys.